Dockerfile

```diff
@@ -50,4 +50,4 @@ COPY --from=compress /gomplate-slim /gomplate
 
 ENTRYPOINT [ "/gomplate" ]
 
-CMD [ "--help" ]
\ No newline at end of file
+CMD [ "--help" ]
```

Comment:
unrelated to this PR

Reply:
wop, don't know how this arrived to the PR